R/utilities-plotting.R

```diff
@@ -683,9 +707,74 @@
 
 .updateSpecificSetting <- function(specificSetting, specificPlotConfiguration, generalPlotConfiguration) {
   if (!is.null(specificPlotConfiguration[[specificSetting]]) &
-    !is.null(generalPlotConfiguration[[specificSetting]])) {
+      !is.null(generalPlotConfiguration[[specificSetting]])) {
     if (generalPlotConfiguration[[specificSetting]] != specificPlotConfiguration[[specificSetting]]) {
       specificPlotConfiguration[[specificSetting]] <- generalPlotConfiguration[[specificSetting]]
     }
   }
 }
+
+#' Names of aggregation available for .extractAggregatedSimulatedData
+#'
+#' @keywords internal
```

Comment:
Let's call it `DataAggregationMethods`, as the aggregation can also be applied to the observed data.

Reply:
After this it can be merged